src/libs/ReportActionsUtils.ts

```diff
@@ -635,6 +665,138 @@ function isNotifiableReportAction(reportAction: OnyxEntry<ReportAction>): boolea
     return actions.includes(reportAction.actionName);
 }
 
+function getMemberChangeMessageElements(reportAction: OnyxEntry<ReportAction>): MemberChangeMessageElement[] {
+    const isInviteAction = isInviteMemberAction(reportAction);
+    const verb = isInviteAction ? Localize.translateLocal('workspace.invite.invited') : Localize.translateLocal('workspace.invite.removed');
+
+    const messageElements: MemberChangeMessageElement[] = [
+        {
+            kind: 'text',
+            content: `${verb} `,
+        },
+    ];
+
+    const originalMessage = reportAction?.originalMessage as ChangeLog;
+    const targetAccountIDs: number[] = originalMessage?.targetAccountIDs ?? [];
+    const personalDetails = PersonalDetailsUtils.getPersonalDetailsByIDs(targetAccountIDs, 0);
+
+    const mentions = targetAccountIDs.map((accountID) => {
+        const personalDetail = personalDetails.find((personal) => personal.accountID === accountID);
+
+        if (personalDetail) {
+            const displayNameOrLogin =
+                LocalePhoneNumber.formatPhoneNumber(personalDetail.login ?? '') || (personalDetail?.displayName ? personalDetail?.displayName : Localize.translateLocal('common.hidden'));
+            return {content: `@${displayNameOrLogin}`, accountID};
+        }
+        return {content: `@${Localize.translateLocal('common.hidden')}`, accountID};
+    });
+
+    const lastMention = mentions.pop();
+    if (!lastMention) {
+        return [];
+    }
+
+    if (mentions.length === 0) {
+        messageElements.push({
+            kind: 'userMention',
+            content: lastMention.content,
+            accountID: lastMention.accountID,
+        });
+    } else if (mentions.length === 1) {
+        messageElements.push(
+            {
+                kind: 'userMention',
+                content: mentions[0].content,
+                accountID: mentions[0].accountID,
+            },
+            {
+                kind: 'text',
+                content: ` ${Localize.translateLocal('common.and')} `,
+            },
+            {
+                kind: 'userMention',
+                content: lastMention.content,
+                accountID: lastMention.accountID,
+            },
+        );
+    } else {
+        mentions.forEach((mention) => {
+            messageElements.push(
+                {
+                    kind: 'userMention',
+                    accountID: mention.accountID,
+                    content: `${mention.content}`,
+                },
+                {
+                    kind: 'text',
+                    content: `, `,
```

Comment:
We can move the space to the next element, and add a key `common.serialComma` which translates to `','`(a comma character) in English and `''` (empty string) in Spanish.

Reply:
I found a better place for this thing. You can cherry-pick this commit:

https://github.com/cubuspl42/App/commit/465a23c99f0fcff797465ee1a280956aa2ef87e3

Reply:
You're right, it will be complex to access this functionality this way. I'll give it another shot.

Reply:
I see the comma before `and` was removed with the ES language

Reply:
Not sure in the ES language we will remove all comma `,` or only before `and`

Reply:
Ok, got it so it work well

Reply:
yes, Thanks for the help, I also just learned about this. I will apply and push the new code.